warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RegexFunctionVisitor.java

```diff
@@ -61,43 +65,75 @@ public Object visit(ASTFunctionNode node, Object data) {
         if (functionMetadata.name().equals(INCLUDE_REGEX) || functionMetadata.name().equals(EXCLUDE_REGEX)) {
             List<JexlNode> arguments = functionMetadata.args();
 
+            if (arguments.get(0) instanceof ASTAndNode) {
+                // without this check we would have changed the AND to an OR
+                log.warn("filter:includeRegex(FIELD1 && FIELD2...) is currently not supported for rewrite");
```

Comment:
I an now wondering if it is more likely that we would hit this case with an EXCLUDE_REGEX?  Just check to make sure that the model expansion is not putting an AND node in here for an EXCLUDE_REGEX.  If it is then we need to handle it.

Reply:
I thought the model expansion was always ORs.  Will verify.  Thanks!

Reply:
https://github.com/NationalSecurityAgency/datawave/blob/integration/warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryModelVisitor.java#L602

Looks like it'll always be ORed in the exludeRegex.... should we fix that in a separate ticket?

Reply:
thinking to just handle it...